index.js

```diff
@@ -1 +1,15 @@
 // Your code here
+const findTime = () => { 
+    const date = new Date;
+    const hours = date.getHours()
+    const minutes = date.getMinutes()
+    const seconds = date.getSeconds()
+
+    const formattedHours = (hours < 10 ? '0' : '') + hours
```

Comment:
I like how you:

a) were explicit with your variable names and 
b) added conditionals for formatting

Overall, very clean and concise!  Great job! :)